features/step_definitions/menu_steps.rb

```diff
@@ -13,6 +13,12 @@
   end
 end
 
+Then /^I should see there is current API/ do
```

Comment:
We should describe behaviors not procedures. `The current API is visible` vs. `I should see the current API`.

Reply:
Hm, that article stresses de difference between describing **behavior** and **implementation**, but I think `The current API is visible` and `I should see the current API` are the same thing (behavior).

Maybe there is another style guide then, that shows whether the behavior should be described from the user point or view, or just an impersonal statement.

Reply:
>One way to make scenarios easier to maintain and less brittle is to use a declarative style. Declarative style describes the behaviour of the application, rather than the implementation details.

It's a matter o being consistent and use less of "I click and there, I..., I..." in favor of more declarative behavior. True in this particular case it's the same.